src/context/directory/handlers/branding.ts

```diff
@@ -10,17 +10,24 @@ type ParsedBranding = ParsedAsset<'branding', Asset>;
 
 function parse(context: DirectoryContext): ParsedBranding {
   const brandingDirectory = path.join(context.filePath, constants.BRANDING_DIRECTORY);
+  const brandingFile = path.join(brandingDirectory, 'branding.json');
 
   if (!existsMustBeDir(brandingDirectory)) return { branding: null };
 
-  const branding = loadJSON(path.join(brandingDirectory, 'branding.json'), context.mappings);
+  const branding = (() => {
```

Comment:
**Idea:** What do you think about renaming this `brandingSettings`? That way it's very clear what this object contains. Would require a couple other lines to change, but I think it would solve the original confusion as to why this bug was introduced in the first place.

Reply:
makes sense and it clears up any confusion around the naming.